vtex-apps/order-shipping
# Add context function to update user's selected address

#### What problem is this solving?
Add a function to update the selected address. This will be used in the `checkout-shipping` step to alter the currently selected address to another pre-registered address.

The scenario is the following:

<img width="615" alt="image" src="https://user-images.githubusercontent.com/10223856/79783181-a50f8b80-8316-11ea-9632-38ab04c76e37.png">

#### How should this be manually tested?
You can access [this workspace](https://lucas--checkoutio.myvtex.com/cart/add?sku=292), but note that the shipping step is still a WIP and we will open a PR for it later.

#### Checklist/Reminders


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes

This PR also includes the changes in #8
